keps/prod-readiness/sig-network/4004.yaml

```diff
@@ -4,3 +4,5 @@
 kep-number: 4004
 alpha:
   approver: "@wojtek-t"
+beta:
+  approver: "@wojtek-t"
```

Comment:
Few comments about the PRR

> Are there any tests for feature enablement/disablement?

Has the manual test mentioned there happened? If so, can you mention it?

> Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?

This isn't N/A - I would expect some manual test to happen to verify if the Node is appropriately updated.

> Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?

This is effectively a deprecation.


Also fill-in troubleshooting section based on template instructions.

Reply:
Thanks for the due diligence review, have added the relevant content.